src/presentation/ui/component/form/fileUploadTextInputFileContentReader.templ

```diff
@@ -0,0 +1,65 @@
+package componentForm
+
+script FileUploadTextInputFileContentReaderLocalState() {
+	document.addEventListener('alpine:init', () => {
+		Alpine.data('fileUploadTextInputFileContentReader', () => ({
+			uploadedFileName: '',
+			get uploadedFileNameLabel() {
+				if (this.uploadedFileName == '') {
+					return 'No file chosen';
+				}
+
+				return this.uploadedFileName;
+			},
+			init() {
+				this.uploadedFileName = '';
+			},
+			handleFileUpload(event) {
+				const inputFiles = Array.from(event.target.files);
+				if (inputFiles.length == 0) {
+					return;
+				}
+
+				const uploadedFile = inputFiles[0];
+				this.uploadedFileName = uploadedFile.name;
+
+				const reader = new FileReader();
+				reader.onload = (event) => {
+					this.$dispatch('file-content-readed', event.target.result);
+				};
+				reader.readAsText(uploadedFile);
+			}
+		}));
+	});
+}
+
+templ FileUploadTextInputFileContentReader(
+	id, label, bindValuePath, acceptAttr string,
+	denseMode bool,
+) {
+	<!-- FileUploadTextInputFileContentReader -->
+	@FileUploadTextInputFileContentReaderLocalState()
+	<div class="relative w-full" x-data="fileUploadTextInputFileContentReader">
+		<input
+			x-ref={ id + "fileInput" }
```

Comment:
Your F here should be uppercase. Otherwise it will be idfileInput instead of idFileInput. I know id will be replaced, but will be replaced by something camelCase.

Reply:
Done!